tests/test_env.py

```diff
@@ -17,16 +17,12 @@
 from rasterio.rio.main import main_group
 from rasterio.session import AWSSession, DummySession, OSSSession, SwiftSession, AzureSession
 
+from .conftest import credentials
 
-# Custom markers.
-credentials = pytest.mark.skipif(
-    not(boto3.Session().get_credentials()),
-    reason="S3 raster access requires credentials")
 
-
-L8TIF = "s3://landsat-pds/L8/139/045/LC81390452014295LGN00/LC81390452014295LGN00_B1.TIF"
-L8TIFB2 = "s3://landsat-pds/L8/139/045/LC81390452014295LGN00/LC81390452014295LGN00_B2.TIF"
-httpstif = "https://landsat-pds.s3.amazonaws.com/L8/139/045/LC81390452014295LGN00/LC81390452014295LGN00_B1.TIF"
+L8TIF = "s3://usgs-landsat/collection02/level-2/standard/oli-tirs/2022/139/045/LC08_L2SP_139045_20220113_20220123_02_T1/LC08_L2SP_139045_20220113_20220123_02_T1_SR_B1.TIF"
+L8TIFB2 = "s3://usgs-landsat/collection02/level-2/standard/oli-tirs/2022/139/045/LC08_L2SP_139045_20220113_20220123_02_T1/LC08_L2SP_139045_20220113_20220123_02_T1_SR_B2.TIF"
```

Comment:
we could also use https://registry.opendata.aws/sentinel-2-l2a-cogs/ which are fully public

Reply:
Thanks @vincentsarago :+1: